java/core/src/java/org/apache/orc/impl/MemoryManagerImpl.java

```diff
@@ -81,6 +81,7 @@ public Thread getOwner() {
    */
   public MemoryManagerImpl(Configuration conf) {
     double maxLoad = OrcConf.MEMORY_POOL.getDouble(conf);
+    ROWS_BETWEEN_CHECKS = OrcConf.ROWS_BETWEEN_CHECKS.getLong(conf);
```

Comment:
validate values here? and set default value in case of invalid values and maybe log?